concent_api/concent_api/tests/test_check_siginig_service_key_availability_for_middleman.py

```diff
@@ -0,0 +1,44 @@
+from copy import deepcopy
+
+from assertpy import assert_that
+from django.conf import settings
+from django.test import override_settings
+
+from common.testing_helpers import generate_ecc_key_pair
+from concent_api.system_check import check_signing_service_key_availability_for_middleman
+from concent_api.system_check import create_error_43_signing_service_public_key_is_missing
+from concent_api.system_check import create_error_44_signing_service_public_key_is_invalid
+
+
+(_, SIGNING_SERVICE_PUBLIC_KEY) = generate_ecc_key_pair()
+
+
+class TestCheckSigningServiceKeyAvailabilityForMiddleMan:  # pylint: disable=no-self-use
+    def test_that_if_middleman_is_disabled_signing_service_public_key_does_not_have_to_be_defined_in_settings(self):
+        with override_settings(CONCENT_FEATURES=deepcopy(settings.CONCENT_FEATURES)):
```

Comment:
I think that `deepcopy` is not needed here. If you modify settings while in `override_settings` context, they shouldn't be changed outside. At least `del` works this way.

Reply:
Done